reconcile/utils/saasherder.py

```diff
@@ -1451,26 +1450,7 @@ def get_saas_targets_config(self, saas_file):
 
     @staticmethod
     def _get_pipelines_provider(saas_file):
-        """Returns the Pipelines Provider for a SaaS file.
-
-        Args:
-            saas_file (dict): SaaS file GQL query result with apiVersion key
-
-        Returns:
-            dict: Pipelines Provider details
-        """
-        saas_file_api_version = saas_file["apiVersion"]
-        if saas_file_api_version == "v1":
-            # wrapping the instance in a pipelines provider structure
-            # for backwards compatibility
-            pipelines_provider = {
-                "provider": Providers.JENKINS,
-                "instance": saas_file["instance"],
-            }
-        if saas_file_api_version == "v2":
-            pipelines_provider = saas_file["pipelinesProvider"]
-
-        return pipelines_provider
+        return saas_file["pipelinesProvider"]
```

Comment:
If we're going to remove the docstring, which probably makes sense given the size of the method, then worth moving the type info here to a type hint?

Reply:
Absolutely! :+1: 

Fixed